tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py{27,35}-django{18,19}, quality
```

Comment:
You're adding py3 support here, which is great, but it doesn't look like you're actually running the tests on Travis with py3.

We should presumably add this test case to `.travis.yml` to ensure it is always run and we don't inadvertently introduce mistakes.

https://tox.readthedocs.org/en/latest/config.html#confval-basepython=NAME-OR-PATH